torch/distributed/_shard/sharding_spec/chunk_sharding_spec_ops/_common.py

```diff
@@ -226,170 +225,13 @@ def _result_distribute_with_col_rearrange(
     return output.index_select(0, torch.tensor(indices, device=output.device))
 
 
-def _handle_row_wise_lookup_distribute(
```

Comment:
does it mean that rowwise look up is not needed anymore?

Reply:
Yeah this is old logic. We don't need this anymore so that we can get rid of unnecessary collective communications.